<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cPrint</class>
 <widget class="QWidget" name="cPrint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
     <item>
      <widget class="QSplitter" name="m_lpSplitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Printer</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="1" column="3">
               <widget class="QCheckBox" name="m_lpPrintToFile">
                <property name="text">
                 <string>Print to file</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="m_lpPrinterSelectLabel">
                <property name="text">
                 <string>Name:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="m_lpPrinterStatus">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="2" colspan="2">
               <widget class="QLabel" name="m_lpPrinterMakeAndModel">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QComboBox" name="m_lpPrinterSelect"/>
              </item>
              <item row="0" column="3">
               <widget class="QPushButton" name="m_lpPrinterProperties">
                <property name="text">
                 <string>Properties</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="m_lpPrinterMakeAndModelLabel">
                <property name="text">
                 <string>Printer Model:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="m_lpPrinterStatusLabel">
                <property name="text">
                 <string>Status:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="2" colspan="2">
               <widget class="QLabel" name="m_lpPrinterLocation">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="m_lpPrinterLocationLabel">
                <property name="text">
                 <string>Location:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Paper</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="m_lpPaperSizeLabel">
                <property name="text">
                 <string>Paper Size:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="m_lpPaperSize"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="m_lpPaperSourceLabel">
                <property name="text">
                 <string>Paper Source:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="m_lpPaperSource"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="m_lpZoomOut">
         <property name="toolTip">
          <string>zoom out</string>
         </property>
         <property name="statusTip">
          <string>zoom out</string>
         </property>
         <property name="text">
          <string>zoom out</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_lpZoomFit">
         <property name="toolTip">
          <string>zoo to fit</string>
         </property>
         <property name="statusTip">
          <string>zoom to fit</string>
         </property>
         <property name="text">
          <string>zoom to fit</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_lpZoomOriginal">
         <property name="toolTip">
          <string>zoom original</string>
         </property>
         <property name="statusTip">
          <string>zoom original</string>
         </property>
         <property name="text">
          <string>zoom original</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_lpZoomIn">
         <property name="toolTip">
          <string>zoom in</string>
         </property>
         <property name="statusTip">
          <string>zoom in</string>
         </property>
         <property name="text">
          <string>zoom in</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_lpPrintPreview">
         <property name="toolTip">
          <string>print preview</string>
         </property>
         <property name="statusTip">
          <string>print preview</string>
         </property>
         <property name="text">
          <string>print preview</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_lpPrint">
         <property name="toolTip">
          <string>print</string>
         </property>
         <property name="statusTip">
          <string>print</string>
         </property>
         <property name="text">
          <string>print</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
